.github/workflows/ci-instrumentation-with-services.yml

```diff
@@ -46,6 +46,7 @@ jobs:
           ruby: "3.1"
           yard: true
           rubocop: true
+          coverage: true
```

Comment:
Tests for the gems in contrib that aren't specifically instrumentation (propagators, resources,  etc.) are run in the `.github/workflows/ci-contrib.yml`. Could you add `coverage: true` to that workflow as well?

Reply:
@kaylareopelle done!